src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -170,6 +170,12 @@ public function remove($files)
                     throw new IOException(sprintf('Failed to remove symlink "%s": ', $file).self::$lastError);
                 }
             } elseif (is_dir($file)) {
+                // Rename directory before removing it to mitigate the chance of
+                // another process writing to it during the operation.
+                // @see https://github.com/symfony/symfony/issues/27578
```

Comment:
should be removed: we don't link to github in the source

Reply:
Nice job for fabbot!?

cc @fabpot